### PR TITLE
Ref: Refactor the quiz session UI

### PR DESCRIPTION
**Pull request by HaroldCuico**
Changes Made
* Added redirection from the Quiz Master to the Quiz session UI
- If user is already logged in in the quiz master it does need to log in again in the session UI
- Passed the token and username to the session UI
* Added redirection from the quiz session to the Quiz Master UI if quiz has ended
* Remove header during quiz

### CODE REVIEW

File: WebApp/frontend/quiz-master/app/dashboard/game.tsx
Comment (jaymar921) [inline WebApp/frontend/quiz-master/app/dashboard/game.tsx:1]: nice, so we are now passing token and name in the query

**Review by jaymar921 — APPROVED** [review]
looks good, approved

File: WebApp/frontend/quiz-master/app/dashboard/game.tsx
Comment (HaroldCuico) [reply WebApp/frontend/quiz-master/app/dashboard/game.tsx:1]: I do need to encrypt the token when I pass the token for security

**Review by kinloveko — APPROVED** [review]
Looks good!

**Pull request merged**